src/runtime/composables/visual-editing.ts

```diff
@@ -132,9 +139,19 @@ const createSanityHelper = (
         queryStore = createQueryStore(visualEditing, client)
       }
     },
+    visualEditing: {
+      isPreviewing: visualEditingState,
+      inFrame: isInFrame(),
+    },
```

Comment:
What do you think about splitting this out into a separate composable?

`useSanityVisualEditingState` or something shorter(?)

Reply:
we could also wrap this in a `reactive` so we don't have to access it via `visualEditing.isPreviewing.value` but it would be kept up to date when the value changes, and accessed just via `visualEditing.isPreviewing` (just like `inFrame`)

Reply:
Yes, I think that would be a much nicer approach. I must admit I ended up attaching this to the helper to skirt the issue of importing a conditionally available composable, as that approach led to type checks failing.

Reply:
ahh, I see. I'll push.